src/browser/api/window.js

```diff
@@ -1956,7 +1956,7 @@ function handleForceActions(identity, force, eventType, eventPayload, defaultAct
 
 
 function applyAdditionalOptionsToWindow(browserWindow) {
-    let options = browserWindow && browserWindow._options;
+    const options = browserWindow && JSON.parse(JSON.stringify(browserWindow._options));
```

Comment:
```suggestion
    const options = browserWindow && Object.assign({}, browserWindow._options);
```

Reply:
`JSON.parse(JSON.stringify(undefined))` raises an exception.

Reply:
I agree with you about a possible exception thrown, but it's not expected, because we set `._options` just 2 lines above: https://github.com/HadoukenIO/core/blob/3147096ffa3c353d442058a152d50f24f75d0924/src/browser/api/window.js#L497

I can't use `Object.assign` as I need a deep clone.

Reply:
Can we just double check using getters / setters to log out access as this is async essentially and maybe, just maybe, something is getting in between and tripping us up here.